docs/rules/no-private-routing-service.md

```diff
@@ -0,0 +1,46 @@
+# no-private-routing-service
+
+Disallow the use of the private `-routing` service.
+
+There has been a public `router` service since Ember 2.16 and using the private routing service should be unnecessary.
+
+## Examples
```

Comment:
Can you add a `## References` section at the bottom with links to the router service spec and any other relevant links to find more information about this?